vms/platformvm/blocks/block.go

```diff
@@ -45,3 +46,11 @@ func initialize(blk Block) error {
 	}
 	return blk.initialize(bytes)
 }
+
+func Size(blk Block) int {
+	if blk == nil {
+		return wrappers.LongLen
```

Comment:
nit: comment where this `Long` comes from

Reply:
comes from the codec - pushed up a comment for the function

Reply:
_actually_ - this was added for pointer overhead... Because we didn't want `nil` blocks to have `0` size (as that's an OOM waiting to happen).